src/services/nacl/connections.ts

```diff
@@ -19,9 +20,38 @@ export default ({
   service: RawAwsNetworkAcl
 }): { [key: string]: ServiceConnection[] } => {
   const connections: ServiceConnection[] = []
-  const { NetworkAclId: id, VpcId: NaclVpcId } = nacl
+  const {
+    NetworkAclId: id,
+    VpcId: NaclVpcId,
+    Associations: naclSubnetAssociations,
+  } = nacl
 
-  // TODO: Add subnet connection
+  const subnetIds = naclSubnetAssociations.map(({ SubnetId }) => SubnetId)
+
+  /**
+   * Find related Subnets
+   */
+  const subnets: {
+    name: string
+    data: { [property: string]: RawAwsSubnet[] }
+  } = data.find(({ name }) => name === services.subnet)
+  if (subnets?.data?.[region]) {
+    const subnet: RawAwsSubnet = subnets.data[region].find(
```

Comment:
could it be associated to more than one subnet? This would only find the first one in the list that matched.

Reply:
Fixed!